addons/bus/static/src/js/web_client_bus.js

```diff
@@ -0,0 +1,92 @@
+odoo.define('bus.WebClient', function (require) {
+"use strict";
+
+var core = require('web.core');
+var WebClient = require('web.WebClient');
+
+var _t = core._t;
+
+WebClient.include({
+    /**
+     * Detects the presence of assets in DOM's HEAD
+     *
+     * @override
+     */
+    start: function () {
+        var self = this;
+        this.assetsChangedNotificationId = null;
+        this._assets = {};
+
+        return this._super.apply(this, arguments).then(function () {
+            var $assets = $('*[data-asset-xmlid]');
+            $assets.each(function () {
+                self._assets[$(this).data('asset-xmlid')] = $(this).data('asset-version');
+            });
+        });
+    },
+    /**
+     * Assign handler to bus notification
+     *
+     * @override
+     */
+    show_application: function () {
+        this.call('bus_service', 'onNotification', this, this._onNotification);
+        this.call('bus_service', 'startPolling');
+        return this._super.apply(this, arguments);
+    },
+    /**
+     * Displays one notification on user's screen when assets have changed
+     *
+     * @private
+     */
+    _displayBundleChangedNotification: function () {
+        var self = this;
+        if (!this.assetsChangedNotificationId) {
+            this.assetsChangedNotificationId = this.call('notification', 'notify', {
+                title: _t('Update available'),
+                message: _t('Refresh your browser to take advantage of the latest update of Odoo in your browser'),
```

Comment:
not sure about the sentence: people should not believe they will be upgraded by pressing f5, obviously, but I don't like the 'in your browser' occurring twice)

Reply:
I'm not sure either of what the message should be to avoid mistaking this refresh with an upgrade of Odoo version
a consult of @odoongr is welcome